tutorials/networking/high_level_multiplayer.rst

```diff
@@ -103,388 +101,280 @@ for full IPv6 support.
 Initializing the network
 ------------------------
 
-The object that controls networking in Godot is the same one that controls everything tree-related: :ref:`SceneTree <class_SceneTree>`.
-
-To initialize high-level networking, the SceneTree must be provided a NetworkedMultiplayerPeer object.
+High level networking in Godot is managed by the :ref:`SceneTree <class_SceneTree>`.
 
-To create that object, it first has to be initialized as a server or client.
+Every node has a ``multiplayer`` property, which is a reference to a default ``MultiplayerAPI`` instance.
```

Comment:
Misleading, unless defined for the sub-tree it defaults to the default one, but this implies it's always a default one